routes/inertia.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Same question here, not sure how this relates to your feature

Reply:
All 3 of these were me accidentally adding in some backend files from a different feature. I removed them in them most recent commit. Thanks for pointing them out.